Integrations/integration-ESM-v10.yml

```diff
@@ -1151,7 +1151,12 @@ script:
 
     except Exception as ex:
         demisto.error('#### error in McAfee ESM v10: ' + str(ex))
-        return_error(str(ex))
+        if demisto.command() == 'fetch-incidents':
+            LOG(str(ex))
```

Comment:
print the whole stack to LOG, or lets support Exception stack trace in logs.
If exception passed to LOG or return_error then print the stack trace to the demisto.debug